tests/functional/arxiv/fixtures/arxiv_smoke_record.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
There's a mixture of tabs and spaces in this file, use only one of them (not a breaking change, but annoying).
Probably spaces is fittest, as all the python files around use them already.